features/custom-user-agent.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_meta": {
+        "description": "Custom user agents used by our native apps that override the default user-agent string.",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "Site Breakage"
+        }
+    },
+    "settings": {
+        "omitApplicationSites": [
+            "cvs.com",
+            "sovietgames.su",
+            "accounts.google.com",
+            "facebook.com"
```

Comment:
hm, wait wait, I just realized that those don't match `sampleExcludeRecords`. We should have domain/reason here, right?